tdr-entrypoint.sh

```diff
@@ -0,0 +1,237 @@
+#!/bin/bash
```

Comment:
Did you write this file from scratch, or is it copied from the main Keycloak project?

If it's the latter, how do we keep it in sync with the Keycloak entry point if they update their copy?

Reply:
It is copied from the supplied entrypoint script. The only way to keep up to date I can think of is to write another script to add the necessary changes for TDR to the supplied entrypoint script. That script would need to be run at the docker build stage.

Reply:
After further discussion and investigation:

- Unable to find a sensible way to override the creation of the master realm at start up other than the one proposed. 
- The docker-entrypoint.sh resets the SYS_PROPS to "" nothing, therefore cannot set these with the additional settings needed to import the master realm json prior to calling this script.

Reply:
That's a shame. Thanks for looking into it!

So does that mean this entrypoint script needs to stay in this repo? If so, can you:

- Add a comment to the top of the file explaining where it came from (including a link), why it's necessary, and what to do if the original gets updated
- Add the original code's licence to the top of the file as well
- Add an "Upgrading Keycloak" section to the readme saying how to get the new version of this file and merge your changes into it